v1/README_Cloud_PubSub_to_Datadog.md

```diff
@@ -40,7 +40,7 @@ on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplat
 * **apiKey** : The Datadog API key. You must provide this value if the `apiKeySource` is set to `PLAINTEXT` or `KMS`. For more information, see API and Application Keys (https://docs.datadoghq.com/account_management/api-app-keys/) in the Datadog documentation.
 * **batchCount** : The batch size for sending multiple events to Datadog. The default is `1` (no batching).
 * **parallelism** : The maximum number of parallel requests. The default is `1` (no parallelism).
-* **includePubsubMessage** : Whether to include the full Pub/Sub message in the payload. The default is `false` (only the data element is included in the payload).
+* **includePubsubMessage** : Whether to include the full Pub/Sub message in the payload. The default is `true` (all elements, including data element, are included in the payload).
```

Comment:
Hey @liferoad we'll already be updating our docs as well as these docs already. We are just concerned that by NOT updating the defaults in the template customers migrating from the previous pub/sub solution would suffer since their log formats would change.  Happy to chat if there are more specific concerns around changing these default values.

Reply:
Can you share the doc link or put this to the readme?

Reply:
Sure thing! Just added.

Reply:
Thanks!